docs/lint_command.md

```diff
@@ -0,0 +1,51 @@
+### Lint
```

Comment:
add an explanation for why do we want this
Make sure that the code is up with python standards, prevents bugs, runs testing, etc...

Reply:
kinda nailed it with this explanation... I had nothing to add... 
Done